libsql/src/replication/mod.rs

```diff
@@ -149,13 +149,13 @@ impl EmbeddedReplicator {
         encryption_config: Option<EncryptionConfig>,
         perodic_sync: Option<Duration>,
     ) -> Result<Self> {
-        let mut replicator =
```

Comment:
Is this done by cargo fmt or you ide? If your ide then please revert this.

Reply:
rust analyzer did this, but it's not doing it if I use cargo fmt outside of analyzer so I reverted it.